core/ui/mpc/keysign/prompt/StartKeysignPromptProps.tsx

```
@@ -1,9 +1,14 @@
-  'securityType'
-> &
-  IsDisabledProp
+  'securityType' | 'keysignPayload'
+>
+
+export type StartKeysignPromptProps =
+  | BaseStartKeysignPromptProps
+  | (
+      | { disabledMessage: string }
+      | Pick<CoreViewState<'keysign'>, 'keysignPayload'>
+      | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)